main.go

```diff
@@ -28,6 +30,18 @@ func Main() {
 	}
 	go cfg.Watch()
 
+	shutdown, err := InitTelemetry(context.TODO(), cfg.Telemetry)
+	if err != nil {
+		log.WithError(err).Error("error creating telemetry")
+	}
+
+	defer func() {
+		log.Info("flushing and shutting down telemetry")
+		if err := shutdown(context.TODO()); err != nil {
```

Comment:
```suggestion
		if err := shutdown(ctx); err != nil {
```
does the Backgound context work here instead of using TODO?